apps/arweave/src/ar_data_sync.erl

```diff
@@ -1510,12 +1518,17 @@ validate_served_chunk(Args) ->
 		false ->
 			case ar_block_index:get_block_bounds(Offset - 1) of
 				{BlockStart, BlockEnd, TXRoot} ->
-					{_Strict, ValidateDataPathFun} =
-						case BlockStart >= ?STRICT_DATA_SPLIT_THRESHOLD of
+					ValidateDataPathFun =
```

Comment:
can this chunk be refactored with the other `ValidateDataPathFun` selection code?